tests/message/test_tasks.py

```diff
@@ -107,22 +107,22 @@ class SubtaskResultsAcceptedTest(mixins.RegisteredMessageTestMixin,
                                  unittest.TestCase):
     FACTORY = factories.tasks.SubtaskResultsAcceptedFactory
     MSG_CLASS = message.tasks.SubtaskResultsAccepted
-    TASK_ID_PROVIDER = 'task_to_compute'
+    TASK_ID_PROVIDER = 'report_computed_task'
 
     def test_factory(self):
         self.assertIsInstance(self.msg, message.tasks.SubtaskResultsAccepted)
 
     def test_task_to_compute_wrong_class(self):
```

Comment:
@dybi please update the name of the test ;)

Reply:
sorry, my bad ;)